draft-packages/menu/KaizenDraft/Menu/MenuDropdown.tsx

```diff
@@ -44,7 +44,17 @@ class MenuDropdown extends React.Component<MenuDropdownProps> {
       const rect = menu.current.getBoundingClientRect()
 
       menu.current.style.bottom =
-        pos.bottom > innerHeight - rect.height ? "24px" : "auto"
+        // If the menu won't fit below the the menu button, show it above instead.
+        // For some reason, a 5px buffer was needed.
+        pos.bottom + 5 > innerHeight - rect.height &&
+        // ...but, do not display it above the menu button, if there's not enough
+        // room, otherwise the user won't even be able to scroll high enough to
+        // see the menu items!
+        rect.top - rect.height - 30 >= 0
+          ? // I'm unsure if this 24px is intentional or not. The menu ends up
+            // overlapping the dropdown menu.
```

Comment:
Side note, would we know if this is a design choice to overlap the menu toggle button or not?

Reply:
> // I'm unsure if this 24px is intentional or not. The menu ends up overlapping the dropdown menu.

I know that the Button is 48px high, so my guess is that the original intention of the 24px was to push the menu down to sit below the button instead of overlapping it halfway.

Reply:
According to the UI Kit, the menu should not overlap. @jeremy8883 If you have time to tweak this so that it doesn't overlap, I'm happy to approve.

![image](https://user-images.githubusercontent.com/6406263/106404370-ac0f2100-6486-11eb-96f3-df1ed319e881.png)

Reply:
No problem! That's fixed now

![image](https://user-images.githubusercontent.com/4495057/106406466-76ba0180-648d-11eb-93f0-08d5b639a425.png)